src/validate.ts

```diff
@@ -0,0 +1,91 @@
+
+'use-strict';
+
+ /**
+ * Generates random request ID
+ */
+
+function generateRequestID(min, max){
```

Comment:
For a given function, the correct style for JSDoc is:

```javascript
/**
 * Description goes here
 * @param paramName Description of parameter
 * @returns The thing that is returned
 */
```

Can we reformat some of these comments?